command/stoker.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/frantjc/sindri/internal/stoker"
 	"github.com/frantjc/sindri/internal/stoker/stokercr"
 	"github.com/frantjc/sindri/internal/stoker/stokercr/controller"
+	"github.com/frantjc/sindri/internal/stoker/stokercr/scanners"
```

Comment:
looks like I misled you a bit--I'd like this to be `github.com/sindri/internal/stoker/stokercr/controller/trivy` and then `trivy.NewScanner`